### ISSUE TITLE
stuck at epoch 0 ,I try to use one or two gpus on single machine, then stucl

### ISSUE THREAD

**Issue by kongzijian** [damo-vilab/AnyDoor#68]
<img width="1326" alt="image" src="https://github.com/ali-vilab/AnyDoor/assets/54501511/8a2db452-a714-4779-9cfa-3561266538e1">
trainging set is :
dataloader = DataLoader(dataset1, num_workers=0, batch_size=batch_size, shuffle=True)
    from pytorch_lightning.callbacks.progress import TQDMProgressBar
    trainer = pl.Trainer(gpus=n_gpus, strategy="ddp", precision=16, accelerator="gpu", callbacks=[TQDMProgressBar(refresh_rate=1)],  accumulate_grad_batches=accumulate_grad_batches,log_every_n_steps=20)

**Comment by XavierCHEN34** [comment]
There may be some bugs for your dataloader. 
We add "try except" in our dataset base.py to skip some samples. You could remove the try except to debug.

**Comment by JoyHuYY1412** [comment]
> <img alt="image" width="1326" src="https://private-user-images.githubusercontent.com/54501511/299592089-8a2db452-a714-4779-9cfa-3561266538e1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAyODYyNjgsIm5iZiI6MTcxMDI4NTk2OCwicGF0aCI6Ii81NDUwMTUxMS8yOTk1OTIwODktOGEyZGI0NTItYTcxNC00Nzc5LTljZmEtMzU2MTI2NjUzOGUxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzEyVDIzMjYwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTllNTE1OWIxMGFjODRhYjU5OTI0Y2VhYTdkZDEzMTgzNDkyYTkzMjJmZmY5ZjM0ZThjNWM3NzVkMjUwMzk2MWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.xu3OlIAYv-_dvk4w8nomLsiSLhPRH6R9M7CuAmJFb0Q"> trainging set is : dataloader = DataLoader(dataset1, num_workers=0, batch_size=batch_size, shuffle=True) from pytorch_lightning.callbacks.progress import TQDMProgressBar trainer = pl.Trainer(gpus=n_gpus, strategy="ddp", precision=16, accelerator="gpu", callbacks=[TQDMProgressBar(refresh_rate=1)], accumulate_grad_batches=accumulate_grad_batches,log_every_n_steps=20)

Hi, did you solve this problem? I met the same problem and thank you for your help.
